public/locales/en.yml

```diff
@@ -73,7 +73,7 @@ en:
     date: Anticipated arrival date
     note: Please include information listed below
     note_additional: |
-      <li><strong>Date of Expected Use:</strong> Special Collections is open by appointment only.  You must schedule a time to view the requested materials through the registration form on the <a href="https://www.lib.utk.edu/special">Special Collections home page</a>.</li>
+      <li><strong>Date of Expected Use:</strong> Collections are stored offsite.  A minimum of 2 full business days are needed to retrieve these items for use.
```

Comment:
```suggestion
      <li><strong>Date of Expected Use:</strong> Collections are stored offsite.  A minimum of 2 full business days are needed to retrieve these items for use.</li>
```

Reply:
@markpbaggett This looks good other than the need to close the list item.

Reply:
@mathewjordan 🤦🏻